src/openrct2/world/Park.cpp

```diff
@@ -532,7 +532,7 @@ money16 Park::CalculateTotalRideValueForMoney() const
         // Add ride value
         if (ride.value != RIDE_VALUE_UNDEFINED)
         {
-            money16 rideValue = static_cast<money16>(ride.value);
+            money64 rideValue = static_cast<money64>(ride.value);
```

Comment:
Isn't this case redundant now?

Reply:
It's still going from a uint16 to an int64